src/app/services/content-management.service.ts

```diff
@@ -1179,12 +1181,21 @@ export class ContentManagementService {
   }
 
   unlockNode(node: NodeEntry) {
-    this.contentApi.unlockNode(node.entry.id).catch(() => {
-      this.store.dispatch(
-        new SnackbarErrorAction('APP.MESSAGES.ERRORS.UNLOCK_NODE', {
-          fileName: node.entry.name
-        })
-      );
-    });
+    this.contentApi.unlockNode(node.entry.id).subscribe(
+      nodeResult => {
+        delete node.entry.properties['cm:lockType'];
+        this.store.dispatch(new SetSelectedNodesAction([node]));
+        if (isPreview) {
+          this.store.dispatch(new ViewFileAction(nodeResult));
```

Comment:
shouldn't store.select(getAppSelection) fire once you dispatch SetSelectedNodesAction?